FluentTerminal.App.ViewModels/TerminalViewModel.cs

```diff
@@ -264,6 +271,11 @@ public TabThemeViewModel BackgroundTabTheme
 
         public ICommand ShowSearchPanelCommand { get; private set; }
 
+        public ICommand IncreaseFontSizeCommand { get; private set; }
+
+        public ICommand DecreaseFontSizeCommand { get; private set; }
```

Comment:
Where do we actually bind these commands to something?

Reply:
We don't need commands in this case since the actions will be handled by xterm.js itself